tests/private/misc.py

```diff
@@ -100,6 +104,11 @@ def test_direct_v2_inbox(self):
         results = self.api.direct_v2_inbox()
         self.assertEqual(results.get('status'), 'ok')
 
+    def test_direct_v2_thread(self):
+        results = self.api.direct_v2_thread()
```

Comment:
```
results = self.api.direct_v2_thread()
```
Pretty sure this is non-functional syntax

Reply:
Is the `thread_id` supposed to be optional? (It's not as implemented above, obviously, but the description of the PR has it in parentheses.)